java/src/main/java/org/whispersystems/signalservice/loki/api/opengroups/PublicChatAPI.kt

```diff
@@ -305,20 +305,22 @@ class PublicChatAPI(userPublicKey: String, private val userPrivateKey: ByteArray
     }
 
     public fun downloadOpenGroupProfilePicture(server: String, endpoint: String): ByteArray? {
-        val actualEndpoint = "/loki/v1/${endpoint.removePrefix("/")}"
-        val url = "$server/${actualEndpoint.removePrefix("/")}"
-        Log.d("Loki", "Downloading open group profile picture from \"$url\".")
-        try {
-            val result = execute(HTTPVerb.GET, server, actualEndpoint).get()
```

Comment:
You can just use this piece of code for avatar downloading in the try block.
`val url = "${server.removeSuffix("/")}/${endpoint.removePrefix("/")}"`
`val outputStream = ByteArrayOutputStream()`
`DownloadUtilities.downloadFile(outputStream, url, FileServerAPI.maxFileSize, null)`
`return outputStream.toByteArray()`